### PR TITLE
e2e/syncer: stop (some of) the bleeding

### PR DESCRIPTION
**Pull request by sttts**
Use a smaller busybox image 1.6GB => 2.66MB 🤡

### CODE REVIEW

**Review by ncdc — APPROVED** [review]

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -145,22 +145,27 @@ jobs:
 
       # Publish a syncer image to the local registry
       - uses: imjasonh/setup-ko@v0.4
-      - run: KO_DOCKER_REPO=localhost:5000 ko publish ./cmd/syncer --insecure-registry -t $(git rev-parse --short "$GITHUB_SHA") > /tmp/syncer-image
+      - run: KO_DOCKER_REPO=${KIND_REGISTRY} ko publish ./cmd/syncer --insecure-registry -t $(git rev-parse --short "$GITHUB_SHA") > /tmp/syncer-image
```
Comment (aojea) [inline .github/workflows/ci.yaml:148]: where is `KIND_REGISTRY` defined?

Comment (sttts) [reply .github/workflows/ci.yaml:148]: from the kind action above I assume

Comment (aojea) [reply .github/workflows/ci.yaml:148]: ic , `container-tools/kind-action@v1`

**Comment by ncdc** [comment]
According to https://hub.docker.com/r/bitnami/kubectl/tags, the compressed size is 46MB. But still, that's big 😄

**Pull request merged**